src/main/java/org/broadinstitute/hellbender/utils/locusiterator/LocusIteratorByState.java

```diff
@@ -33,7 +33,7 @@
  * That is, the pileup is ordered in a way consistent with the SAM coordinate ordering
  * -- Only aligned reads with at least one on-genomic cigar operator are passed on in the pileups.  That is,
  * unmapped reads or reads that are all insertions (10I) or soft clipped (10S) are not passed on.
- * -- LIBS can perform per-sample downsampling of a variety of kinds.
+ * -- LIBS can perform per-sample downsampling of a variety of kinds. TODO: it is true?
```

Comment:
I added this TODO here while doing this changes because I found that the per-sample downsampling is using `LevelingDownsampler`, so I don't know if it is true that it performs dowsampling "of a variety of kinds", even if the `DownsamplingMethod` passed to the constructor is different.